doobie/src/main/scala/tofu/syntax/doobie/txr.scala

```diff
@@ -0,0 +1,19 @@
+package tofu.syntax.doobie
+
+import fs2.Stream
+import tofu.doobie.transactor.Txr
+
+private[doobie] trait TxrSyntax {
+  implicit def toTxrOps[DB[_], A](dba: DB[A]): TxrOps[DB, A]                     = new TxrOps(dba)
+  implicit def toTxrStreamOps[DB[_], A](dba: Stream[DB, A]): TxrStreamOps[DB, A] = new TxrStreamOps(dba)
+}
+
+private[doobie] final class TxrOps[DB[_], A](private val dba: DB[A]) extends AnyVal {
+  def trans[F[_]](implicit txr: Txr.Aux[F, DB]): F[A]    = txr.trans(dba)
```

Comment:
I am a little afraid that such method name shortening would make it harder to understand code at a first glance.
I guess it won't hurt anyone to name it `transact` & `rawTransact`.

Reply:
Same for streams one

Reply:
I took the names from `doobie` to reduce entropy. I don't want to go the ZIO way.

Reply:
Okay, this is legit too.